templates/studentProfile.phtml

```diff
@@ -5,10 +5,12 @@
     require 'partials/layout/head.phtml';
     echo \Portal\ViewHelpers\LayoutViewHelper::outputTitle('Applicants');
     ?>
+    <script>const studentId = <?=intval($id);?></script>
```

Comment:
it would be better to store this as a data attribute somewhere on the page and then retrieve it in the js using dataset